### DISCUSSION TITLE
algebra relazionale - 10

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#67] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/b7561958-a554-4b47-81fc-dff1dead1b32)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-16 09:46:30](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/34f8ca1d-924f-4643-aee4-db71fe821d71)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/8b7b35a6-3104-4941-ae7a-b926df1b8014)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/c244cd15-e4cd-4248-b6cf-602f68ecf45b)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-03 183711](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/ba7ee9ed-cc43-449f-9951-8cb865c05b95)
![Immagine 2024-02-03 183722](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/faf845fc-df5f-4700-ae10-2b3ea6ef887f)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/b8d8d01e-b31a-4f1f-a791-f35ab12644cf)

**Reply by IonutCicio** [reply]
In casi in cui stai facendo il join fra più tabelle, e due di queste hanno lo attributi con lo stesso nome (ad esempio $\text{MAGAZZINO}$ e $\text{PRODOTTO}$) è consigliabile scrivere $\text{MAGAZZINO.Codice = CodiceM}$ e $\text{CodiceP = PRODOTTO.Codice}$

**Reply by Frisbon** [reply]
grazie, lo terró a mente
